inc/views/base/showthread/showthread.twig

```diff
@@ -205,7 +205,7 @@
         </div>
 
         {% if mybb.settings.allowthreadratings and forum.allowtratings %}
```

Comment:
Isn't this a useless check right now? The whole if could be commented

Reply:
The script is still doing the same as what we do in v1.8, fetch/calculate/output thread rating. For the theme engine, it can filter what will/won't show. Seems the team's idea is to preserve exiting data and just redact the thread rating view from front-end, but allow plugins to reuse this data field.

- The most important for contributors would be the Twig templates conversion.
- And we can make sure that the thread rating is deprecated in 1.9, and some modifications to user profile editing section.
- If anything in doubt, preserve what 1.8 has, and accommodate it with 1.9.

Besides, the new templates should have been written in maybe nearly 10 or 11 months ago. The original idea is fully transform templates to use Twig, without considering too much. @euantorano for clarification.